backend/inspirehep/disambiguation/cli.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+import click
+from elasticsearch_dsl import Q
+from inspire_utils.record import get_values_for_schema
+from invenio_db import db
+
+from inspirehep.records.api import AuthorsRecord
+from inspirehep.records.models import RecordsAuthors
+from inspirehep.search.api import AuthorsSearch
+
+
+@click.group()
+def disambiguation():
+    """Commands for disambiguation"""
+
+
+@disambiguation.command()
+def clean_stub_authors():
```

Comment:
I would add some more comments to this function to explain what's going on w.r.t. consistency, as it might look superficially like you're doing useless work (reading from DB data you've already obtained through ES) and you don't have explicit tests for concurrent scenarios (I don't think you should add those, it's probably not worth it).